packages/react-slider/src/components/Slider/Slider.tsx

```diff
@@ -5,13 +5,15 @@ import { renderSlider } from './renderSlider';
 import { useSliderStyles } from './useSliderStyles';
 
 /**
- * Slider component
+ * Define a styled Slider, using the `useSlider` hook.
+ * {@docCategory Slider}
  */
-export const Slider = React.forwardRef<HTMLElement, SliderProps>((props, ref) => {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const Slider = React.forwardRef<HTMLElement, SliderProps>((props, ref: any) => {
```

Comment:
Let's figure out how to have something better than `any` here for the `ref` as we mature this version of the `Slider`.